### ISSUE TITLE
Determine exit strategy for contents of CMS Research Airtable 

### ISSUE THREAD

**Issue by joagnitti** [department-of-veterans-affairs/va.gov-cms#15481]
## User Story or Problem Statement
As a UXR, I want to make sure all previous research, artifacts, and governance elements currently housed within the CMS Research base on Airtable are safely moved to a more centralized platform (e.g., Confluence or Sharepoint). 

## Description or Additional Context 
The current Airtable instance will be deprecated in the near future as the CMS team transitions to the Platform crew. As such, we need to rehouse the contents of this space to a location that is 1) accessible by all necessary parties (e.g., CMS team members, VA stakeholders, etc.) and 2) is approved/supported by VA leadership (i.e., our PO). 

## Steps for Implementation
- [x] Investigate options for new home
- [x] Discuss options with PO 
- [x] Once new home is identified, create ticket to begin the migration work. 

## Acceptance Criteria
- [x] New location has been identified
- [x] Ticket to begin the migration work has been created

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by EWashb** [comment]
I'm supportive of SharePoint or Confluence to house our Airtable Data. A few things to consider:

1. Product Teams do not have access to Confluence, so if they need the data from those sets, we would have to export it each time. Confluence is fine otherwise.
2. SharePoint could be accessible by all. We could also create a bigger SharePoint site if need be. And that could be attached to our CMS Teams channel (I created this and added all the CMS members but you'll have to log in to CAG and look at the Teams chat). I started the space but didn't do anything with it yet.
